requirements/base.txt

```diff
@@ -1,5 +1,6 @@
+setuptools==44.0.0
 git+https://github.com/nilegraddis/fbs@additional_hooks_dir
 PyQt5==5.9.2
 PyInstaller==3.4
-git+https://github.com/alleninstitute/ipfx
-pyqtgraph==0.10.0
+git+https://github.com/alleninstitute/ipfx@ff0614e2ba7ae1fac227c4d2f58b78799430ab78
```

Comment:
can you set @dev instead of this commit (until we deploy to pypi)

Reply:
@kasbaker It might be a good idea to hold off on this for now. We've got to update ipfx to depend on today's pynwb release and [propagate this change to sweep_qc_tool](https://github.com/AllenInstitute/ipfx/issues/415) and it seems like we would be overriding your work.

Reply:
@NileGraddis OK, thanks for the update. If the sweep qc tool needs to be updated to work with this new release I am happy to help.